Jenkinsfile

```
@@ -47,118 +47,144 @@ properties(
     ]
 )
 
-node('ubuntu-18-x64') {
-    stage('Generate Tarball') {
-        cleanWs()
 
-        checkout scm
+pipeline {
 
-        dir(path: 'clamav_documentation') {
-            git(url: 'https://github.com/Cisco-Talos/clamav-documentation.git', branch: "gh-pages")
-        }
+    agent {
+        label "ubuntu-18-x64"
+    }
 
-        dir(path: 'docs/html') {
-            sh '''# Move the clamav-documentation here.
-                cp -r ../../clamav_documentation/* .
-                # Clean-up
-                rm -rf ../../clamav_documentation
-                rm -rf .git .nojekyll CNAME Placeholder || true
-                '''
-        }
+    stages {
 
-        dir(path: 'build') {
-            sh """# CPack
-                cmake .. -D VENDOR_DEPENDENCIES=ON
-                cpack --config CPackSourceConfig.cmake """
-            archiveArtifacts(artifacts: "clamav-${params.VERSION}*.tar.gz", onlyIfSuccessful: true)
+        stage('GitGaurdian Scan') {
+            environment {
+                GITGUARDIAN_API_KEY = credentials('gitgaudian-ravi-token')
```

```suggestion
                GITGUARDIAN_API_KEY = credentials('gitgaudian-ravi-token')
```

We may need a generic account/team account for this token so it doesn't have to be a personal one. 

Also, minor typo "gitgaudian" should be "gitgaurdian".